lib/search/query_components/autocomplete.rb

```diff
@@ -8,8 +8,11 @@ def payload
           "prefix" => search_term,
           "completion" => {
             "field" => AUTOCOMPLETE_FIELD,
-            "size" => 10,
+            "size" => 8,
             "skip_duplicates" => true,
+            "fuzzy" => {
+              "fuzziness" => 2,
```

Comment:
Is there a reason that we're not using `AUTO` for the fuzziness factor?  The docs seem to imply that it's generally advised? https://www.elastic.co/guide/en/elasticsearch/reference/6.7/common-options.html#fuzziness

Reply:
Good catch, amended.

Reply:
(For completion we have to explicitly state the factor, even if it's AUTO, so the line itself remains)